views/confirmation.ejs

```diff
@@ -0,0 +1,30 @@
+<div class="container">
+  <div class="row justify-content-center py-5 text-center">
+    <div class="col-11 col-md-9" style="padding: 5rem 1rem 2rem; background-color: #EEF8F3; border-radius: 15px;">
+      <!-- Checkmark Icon -->
+      <div class="py-5">
+        <a href="/" style="color: #639C70;">
+          <svg xmlns="http://www.w3.org/2000/svg" width="160" height="160" fill="currentColor" class="bi bi-check-circle-fill" viewBox="0 0 16 16">
+            <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-3.97-3.03a.75.75 0 0 0-1.08.022L7.477 9.417 5.384 7.323a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-.01-1.05z"/>
+          </svg>
+        </a>
+      </div>
+      <h1 class="h3 font-weight-bold">Thank you!</h1>
+        <p class="pb-5">We will review your application shortly!</p>
```

Comment:
I know this is from figma but we should change this text since we won't review it. Maybe something like:
_Your application can now be viewed on Green Pick._

Don't have to do this for this sprint though, we have to change the link for "Go back to profile" as soon as we have profiles anyway

Reply:
I agree, it sounds a bit silly. I can change it now too since I have to commit again anyways.